nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/pom.xml

```diff
@@ -52,6 +52,16 @@ language governing permissions and limitations under the License. -->
             <version>1.14.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.1.1-jre</version>
+        </dependency>
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>fastjson</artifactId>
+            <version>1.2.76</version>
```

Comment:
Is there a particular reason for using `fastjson` as opposed to Jackson?  Although fastjson is license under ASL 2.0, Jackson is more commonly used in other components.

Reply:
There's no particular reason. Would you recommend changing it to jackson?

Reply:
Thanks for the reply, yes, changing to use Jackson ObjectMapper should work, and also bring the implementation in line with other processors that handle JSON.